### PR TITLE
fix for flake8 and autopep8

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: benchmarks/functions/__init__.py [hunk 1]
```diff
@@ -47,12 +47,14 @@ def _check_format(self, outputs, grad_outputs):
         grad_outputs_is_list = isinstance(grad_outputs, (list, tuple))
         if outputs_is_list and grad_outputs_is_list:
             if len(outputs) != len(grad_outputs):
-                msg = 'Format of grad_outputs is not same as that of the output of target function'
+                msg = 'Format of grad_outputs is not same as '
```
Comment (asi1024) [inline benchmarks/functions/__init__.py]: Invalid syntax.
correct: `msg = 'Format of grad_outputs is not same as that of the output of target function' \`

File: benchmarks/functions/__init__.py [hunk 2]
```diff
@@ -47,12 +47,14 @@ def _check_format(self, outputs, grad_outputs):
...
+                    'that of the output of target function'
                 raise ValueError(msg)
             for i in range(len(outputs)):
                 self._check_format(outputs[i], grad_outputs[i])
         elif outputs_is_list or grad_outputs_is_list:
-            msg = 'Type of grad_outputs is not same as that of the output of target function'
+            msg = 'Format of grad_outputs is not same as '
```
Comment (asi1024) [inline benchmarks/functions/__init__.py]: ditto.

File: benchmarks/functions/__init__.py (hunk 1 above)
Comment (kmaehashi) [reply benchmarks/functions/__init__.py]: Oh, sorry.
(BTW in Chainer/CuPy we use parenthesis for line continuation instead of `\`.)

**Comment by kmaehashi** [comment]
Thanks for review, fixed accordingly.

**Comment by asi1024** [comment]
LGTM

**Pull request merged**